presets/TestSamples/4.3/rc_smoothing/race_rc_smoothing.txt

```diff
@@ -0,0 +1,19 @@
+# Title: Race RC smoothing
+# FirmwareVersion: 4.3
+# Category: RC_SMOOTHING
+# Official: true
+# Keywords: aggressive, rc, smoothing, race
+# Author: Chris Thompson ctzsnooze
+# Description: RC smoothing settings for racing - assumes a good RC link
```

Comment:
Should we specify what is "good Rc link"? What's the minimum update rate required. Or should we make race presets for different links/update rates?

Reply:
Yes is difficult.
Maybe "CRSFshot or better"?

Reply:
@ctzsnooze  at least something that explains what is "good". Because for some "good RC link" is the one that goes far (crossfire 50hz), not the one with fast update rate and minimum jitter :)

Reply:
This is difficult because is FrSky a good link?  I think so. 
Maybe I mean a fixed rate link.
I think it might be better to just remove `- assumes a good RC link` :-)